redis/src/cmd.rs

```diff
@@ -584,6 +584,22 @@ impl Cmd {
     }
 }
 
+impl fmt::Debug for Cmd {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let res = self
+            .args_iter()
+            .map(|arg| {
+                let bytes = match arg {
+                    Arg::Cursor => b"<scan_cursor>",
```

Comment:
I dont get this line, why `b"<scan_cursor>"` and not the cursor?

Reply:
The cursor is being passed externally and it isn't saved inside the arg, so I can't print it in this stage. Either way, the cursor doesn't have any meaning to the user/client, so skip logging the exact cursor is good enough for debugging commands in the client side IMO.